### PR TITLE
Issue: #466 Created c++ and updated readme file for Minimum insertion to form a palindrome

### PR DESCRIPTION
**Pull request by AastikSharma05**
## Related Issue
- The problem is Given string str, the task is to find the minimum number of characters to be inserted to convert it to a palindrome. (please mention issue number #466 )
- Added the code implementation of Minimum insertions to form a palindrome Problem using dp.
- Added the explanation of Minimum insertions to form a palindrome Problem in the README.md file.
## Proposed Changes
- Added the code of Minimum insertions to form a palindrome Problem using dp.
- Added README.md file of Minimum insertions to form a palindrome Problem with time and space complexity.

## Additional Info
- This code uses dynamic programming.
- This problem is a variation of LCS problem.
- This is one of the commonly asked interview questions and the concept of this question helps us to solve many more problems based on subsets.
- This code solves an open issue #466 
- I have tested these changes on VS code using multiple inputs and matched the output with the expected output.
- This change will not affect or create conflict with any other code
- Time complexity: O(N^2) 
- Space complexity: O(N^2)

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      | Screenshot #2  |
| ----------- | ----------- |
| Readme preview  | Code snippet     |
| Screenshot 2022-03-09 190052  |   Screenshot 2022-03-09 190120   |

### CODE REVIEW

**Comment by AastikSharma05** [comment]
Please review my pr @iamgunjan @miraj0507

File: D-DynamicProgramming/Minimum_insertions_form_palindrome.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	
+	// The output is length of string minus length of lcs of
+	// str and it reverse
+	return (n - lcs(str, rev, n, n));
+}
+
+// Driver code
+int main()
+{
+	string str = "geeks";
```
Comment (siddhi-244) [inline D-DynamicProgramming/Minimum_insertions_form_palindrome.cpp:62]: Can uh take the string as input?

**Comment by AastikSharma05** [comment]
@siddhi-244 Mam I have made the changes. Please tell if anything else needs to be done.

File: D-DynamicProgramming/Minimum_insertions_form_palindrome.cpp (hunk 1 above)
Comment (AastikSharma05) [reply D-DynamicProgramming/Minimum_insertions_form_palindrome.cpp]: done

**Review by Lakhankumawat — APPROVED** [review]
README looks good @siddhi-244  will review the code, thanks for your contribution

**Comment by AastikSharma05** [comment]
@siddhi-244 plz review the code.

**Comment by siddhi-244** [comment]
> @siddhi-244 plz review the code.

Could you resolve the merge conflicts?

**Comment by AastikSharma05** [comment]
@siddhi-244 @Lakhankumawat I have made the necessary changes so please review my pr and tell if anything else needs to be done

**Review by Lakhankumawat — APPROVED** [review]
Your Code looks good , 
Your README looks ok 💯🌟
Keep contributing

**Pull request merged**